### DISCUSSION TITLE
Dishwasher shows as TV

### DISCUSSION THREAD

**Discussion by thestudiojq** [nVuln/homebridge-lg-thinq#281] [Q&A]
I have an LG dishwasher, which shows up and functions correctly in Homebridge and Homekit. However, the main tile is registered as a TV. (It's the same thing whether I add it as an accessory of my main bridge or as a child bridge.) Is there any way to change this?

I understand that there is no dishwasher type and that this is why the secondary tile with the switch shows up as a water valve. This is great. However, the main status tile, which is also the tile that shows if you choose to display as a single tile, is registered as a TV. Could this perhaps be registered as a valve as well? 

This has no effect on functionality. It's just weird seeing it listed under my speakers and TVs.

Side, but related note: The dishwasher does not show in the Eve app. Or else I would have just used that app to change the type of device.

Thanks in advance for any input anyone can provide.

**Comment by JimMeldrum** [comment]
I'm experiencing the same thing. I have an LG Dishwasher, Washer, Fridge and Dryer. Only the Dishwasher shows as a TV.
